Eastern-Research-Group/mywaterway
# Feature/combine tribes sources

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3672373
* https://app.breeze.pm/projects/100762/cards/3672364

## Main Changes:
* Renamed the default "Esri World Geocoder" to "Address, zip code, and place search".
* Combined the Tribes search sources into a single source.
  * I kept running into issues trying to use JavaScript/CSS to accomplish this. I ended up making my own Search widget that utilizes the Esri Search widget to get the suggestions. This will be more flexible and make it easier for us to make changes to this search widget.

## Steps To Test:
1. Type something into the location search
2. Verify the first header says "Address, zip code, and place search" instead of "ArcGIS World Geocoding Service"
3. Type "reserv" in the location search widget
4. Verify the results only displays one "EPA Tribal Areas" header
  * The current version of HMW displays "EPA Tribal Areas - Lower 48 States" and "EPA Tribal Areas - Alaska Reservations" 
5. Click the down arrow to the left of the search widget
6. Verify the following items are visible: "All", "Address, zip code, and place search", and "EPA Tribal Areas". 
7. Test keyboard navigation around the search widget
